examples/onpremise/userAgentClientHints-Web.php

```diff
@@ -56,7 +56,7 @@
  * 
  **/
 
-require(__dir__ . "/../../vendor/autoload.php");
+require(__DIR__ . "/../../vendor/autoload.php");
```

Comment:
do we need this require here?

Reply:
Same as the other examples. They're not part of the package which would be used by client code, so they're not covered by the autoloader.